src/components/organisms/AdminVenueView/components/VenueRoomsEditor/VenueRoomsEditor.tsx

```diff
@@ -21,7 +21,7 @@ import { DragItem } from "pages/Account/Venue/VenueMapEdition/interfaces";
 import { ItemTypes } from "pages/Account/Venue/VenueMapEdition/ItemTypes";
 import { snapToGrid as doSnapToGrid } from "pages/Account/Venue/VenueMapEdition/snapToGrid";
 
-import { CenterContent } from "components/atoms/CenterContent";
+import { MapBackgroundPlaceholder } from "../MapBackgroundPlaceholder";
```

Comment:
This should be using absolute path.

Reply:
fixed in 2bb4a722ddb53f61dcec6f33062a584b8aa8c70c